src/sxastarter/package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Strange, not sure why these changes to `package.json` and `package-lock.json` are showing in the PR. These changes are already on xmcloud-foundation-head-staging:main.